.github/workflows/build_demos.yml

```diff
@@ -0,0 +1,12 @@
+name: Build demos
+
+on:
+  workflow_dispatch:
+  push:
+
+jobs:
+  run-check:
+    runs-on: macos-11
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/demo/macos
```

Comment:
I think we need to discuss which workflows should exist. We could create two workflows for example. One for the demos and one for the libraries.

Reply:
Is it ok if I move it to the two current workflows, like for the Windows demo?

- https://github.com/maplibre/maplibre-rs/blob/7371ae56fd1cf42cd1048cb024995a6228d7c2de/.github/workflows/on_main_push.yml
- https://github.com/maplibre/maplibre-rs/blob/7371ae56fd1cf42cd1048cb024995a6228d7c2de/.github/workflows/on_pull_request.yml